Source/Wpf/Prism.Wpf/Properties/AssemblyInfo.cs

```diff
@@ -24,5 +24,6 @@
 [assembly: XmlnsDefinition("http://prismlibrary.com/", "Prism.Interactivity")]
 [assembly: XmlnsDefinition("http://prismlibrary.com/", "Prism.Interactivity.InteractionRequest")]
 [assembly: XmlnsDefinition("http://prismlibrary.com/", "Prism.Services.Dialogs")]
+[assembly: XmlnsDefinition("http://prismlibrary.com/", "Prism.Common")]
```

Comment:
do not want to expose Common in XAML

Reply:
As you said, the right location for the attached property is under Prism.Services.Dialogs, so Prism.Common will not be exposed.